src/core/friendly_errors/fes_core.js

```diff
@@ -1,79 +1,6 @@
 /**
  * @for p5
  * @requires core
- *
- * This is the main file for the Friendly Error System (FES). Here is a
```

Comment:
is the intent that markdown file for the contributor docs replaces this? might be nice to link from here somehow, if possible?

Reply:
I thought (1) this information would be easier to read with some markdown formatting and (2) the `fes_core.js` file was getting quite long and would benefit from modularization/organization (maybe for future work!). 
I agree this will be a great place to add a link to the fes_reference_dev_notes.md. 👍